index.html

```diff
@@ -56,6 +56,25 @@ <h3><a href="#">physionic</a></h3>
         </div>
     </header>
     <!-- ========== End hero ========== -->
+    <!-- ========== Start newsletter ========== -->
+    <div class="newsletter">
+        <div class="container flex-container">
+            <div class="newsletter__header section__header">
+                <h4 class="section__title newsletter__title">
+                    get in touch
+                </h4>
+                <div class="newsletter__desc section__desc">
+                    The gradual accumulation of informYation about atomic and
+                    small-scale behaviour during the first quarter of the 20th
+                </div>
+            </div>
+            <div class="newsletter__subscribe">
+                <input type="email" name="subscribe" id="subscribe" placeholder="Your Email">
```

Comment:
it's always prefered to use a label element with your input fields, even if it would be empty, it helps for accessibility and more:

![image](https://user-images.githubusercontent.com/44810632/154512238-7d78581a-bfea-4935-adfe-631bccb40d79.png)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label